libgpos/include/gpos/task/CAutoTaskProxy.h

```diff
@@ -58,7 +58,7 @@ namespace gpos
 			CEvent m_event;
 
 			// propagate error of sub-task or not
-			BOOL m_fPropagateError;
+			BOOL m_fPropagateSubTaskError;
```

Comment:
I spot an inconsistency between comments and code. The comment said Error including `sub-task or not`, but with the rename, this error is only limited to `sub-task`.

Question is what's NOT `sub-task` error? Any example?